src/cloud/docker/docker-containers-cli.md

```diff
@@ -51,83 +50,86 @@ These commands are available in any of the containers.
 
-This container is used for the cronjob, it runs the scheduled cronjobs and can also be used to do one off cron runs.
+The cron container runs the scheduled cron jobs automatically. You can also use it to run cron jobs manually.
 
-#### Container Usage
-
-The Cron container is based on PHP-CLI images, and executes operations in the background immediately after the Docker environment start. It uses the cron configuration defined in the [`crons` property of the `.magento.app.yaml` file]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#crons). To view the cron log:
+{:.procedure}
+To view the cron log:
 
 ```bash
 docker-compose run deploy bash -c "cat /app/var/cron.log"
 ```
 
--  The `setup:cron:run` and `cron:update` commands are not available on Cloud and Docker for Cloud environment
+-  The `setup:cron:run` and `cron:update` commands are not available on Cloud and Docker for Cloud environments
 -  Cron only works with the CLI container to run the `./bin/magento cron:run` command
 
-This container has no extra configuration, however if performance problems from cron runs are present you can use the following snippet in your docker-compose.override.yml
+{:.procedure}
+To run cron jobs manually:
 
-```yaml
-  cron:
-    entrypoint: "bash -c"
+```bash
+docker-compose run cron /usr/local/bin/php bin/magento cron:run
 ```
-This disables the cron container from running automatically.
 
-You can then run the cron manually using the following command.
+### Disable the cron container
 
-```bash
-docker-compose run cron /usr/local/bin/php bin/magento cron:run
+If cron runs cause performance problems, you can prevent the cron container from running automatically by adding following snippet to your `docker-compose.override.yml` file.
+
+```yaml
+cron:
+  entrypoint: "bash -c"
 ```
 
-### Deploy Container
+After disabling the cron container, use docker compose to run cron jobs manually.
 
-#### Container Information
+## Deploy container
 
--  Name: deploy
--  Docker Base Image: [magento/magento-cloud-docker-php](https://hub.docker.com/r/magento/magento-cloud-docker-php)
--  Docker Image Based On: [php](https://hub.docker.com/_/php)
+Container name |Docker base image
+-------- | --------
+deploy | [magento/magento-cloud-docker-php], which is based on the [php] Docker image
 
-This container is used for the deploy process. This mimics Magento Cloud behaviour so testing of the build and deploy process is as close to production as possible.
+This deploy container mimics the Magento Cloud deploy process so that testing the build and deploy process is as close to testing in production as possible.
```

Comment:
```suggestion
The deploy container mimics the Magento Cloud deploy process so that testing the build and deploy process is as close to testing in production as possible.
```